content/collections/experiment-apis/en/experiment-management-api-experiments.md

```diff
@@ -1080,7 +1080,7 @@ Edit an experiment.
 | `bucketingUnit`                        | Optional    | string       | Bucketing unit represented by a group type from the accounts add-on. Used for group level bucketing and analysis.                                                                                          |
 | `evaluationMode`                       | Optional    | string       | Evaluation mode for the experiment, either `local` or `remote`.                                                                                                                                            |
 | `rolloutPercentage`                    | Optional    | number       | Rollout percentage for non-targeted users. Range 0 - 100.                                                                                                                                                  |
-| `targetSegments`                       | Optional    | object       | See the [`targetSegments`](#targetsegments) table for more information. When `targetSegments` object is provided, it will replace existing target segments. Note: cohorts are not supported at the moment. |
+| `targetSegments`                       | Optional    | object array      | See the [`targetSegments`](#targetsegments) table for more information. When you provide the `targetSegments` object, it replaces existing target segments. This option doesn't support cohorts. |
```

Comment:
@markzegarelli https://github.com/amplitude/amplitude-docs/blob/main/content/collections/experiment-apis/en/experiment-management-api-flags.md Do we need to make the same change here
<img width="985" height="101" alt="Screenshot 2025-08-01 at 4 34 50 PM" src="https://github.com/user-attachments/assets/b8436ed5-6c86-4714-b228-afa340bd78d9" />